back/db/views/analytics_fact_participations_v06.sql

```diff
@@ -0,0 +1,29 @@
+-- Analytics view for all types of participation based on the activities table
+
+SELECT
+    a.id,
+    a.user_id AS dimension_user_id,
+    COALESCE(a.user_id, a.id) AS participant_id,
```

Comment:
This is used to de-duplicate participations by the same user, while still counting participations by anonymous users individually (even though they all have `user_id: nil`